### DISCUSSION TITLE
What progress has been done in relation to the initial assumptions?

### DISCUSSION THREAD

**Discussion by wrzepecki** [tari-project/tari#3933] [Q&A]
I remember when in 2018 Tari was announced, now we have 2022 and still have not heard much about mainnet release, can anyone say in percentage unit how much have been done in relation to the initial assumptions?

**Comment by delta1** [comment]
What are the "initial assumptions" you're referring to? 

I don't think anyone can give you a percentage that's not entirely made up. There is [no roadmap](https://www.tari.com/faq/#faq-tari-roadmap). 

Please read 
- [Wen Mainnet?](https://www.tari.com/updates/2021-09-28-update-61.html)
- [Tari 2022](https://www.tari.com/updates/2022-01-06-update-67.html)
- [Dibbler testnet is live](https://www.tari.com/dibbler-testnet-is-live/)
- [Testnet hard fork coming up](https://www.tari.com/updates/2022-03-17-update-73.html)

If you're following on Discord or Telegram or IRC then you'll know when the mainnet launch date is announced.
